sunnypilot/selfdrive/car/interfaces.py

```diff
@@ -66,6 +67,24 @@ def _initialize_torque_lateral_control(CI: CarInterfaceBase, CP: structs.CarPara
     CI.configure_torque_tune(CP.carFingerprint, CP.lateralTuning)
 
 
+def _cleanup_unsupported_params(CP: structs.CarParams, CP_SP: structs.CarParamsSP, params: Params = None) -> None:
```

Comment:
Can we carlog as warnings when we are clearing up certain things to leave a breadcrumb for us ? I feel this may be forgotten at some point and we'd be wondering why the heck we see a particular behavior